### ISSUE TITLE
InputNumber - not able to type dot or comma

### ISSUE THREAD

**Issue by DukaN** [primefaces/primeng#10157]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/primeng-inputnumber-demo-7pebpw

**Current behavior**
In inputnumber user try to type dot or comma, then nothing happening.

**Expected behavior**
If in decimal value user put dot or comma, then fraction should be appear.

**Minimal reproduction of the problem with instructions**
In stackblitz case user try to put dot or comma in decimal value then nothing happening. I'm trying diffrent property combination and is exist only one solution. The inputnumber must have minFractionDigits like 1, but in this case user see on start fraction precision. 

**What is the motivation / use case for changing the behavior?**
I think if user want to type fraction should do it after type dot or comma. 

* **Angular version:** 11.0.8

* **PrimeNG version:** 11.3.2

* **Browser:** [all]

**Comment by DukaN** [comment]
Anyone help? On primeng v12 bug still exist.

**Comment by mertsincan** [comment]
Hi,

I think this issue has been fixed in 12.1.1.

For now, closed. If this problem persists, please reopen this ticket.
Best Regards,
